src/install.rst

```diff
@@ -6,197 +6,314 @@ Installing Nextstrain
 
-    conda update -n base conda
+.. tabs::
 
-Install Nextstrain with Conda or Docker
-=======================================
+   .. group-tab:: macOS
 
-Next, decide whether you prefer to install Nextstrain with Conda or Docker.
-We recommend Conda for M1 Mac and Windows users.
-Docker is not yet ready for widespread use on the M1 Mac.
-Similarly, there are still significant obstacles to running Docker with Windows, as documented in `our issue tracking the problems <https://github.com/nextstrain/cli/issues/31>`_.
+      .. note::
 
-.. tabs::
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
 
-   .. group-tab:: Conda
+      1. Install Miniconda:
 
-      Create a Conda environment named ``nextstrain``.
-      This command will install Nextstrain and its dependencies.
+         a. Go to the `installation page <https://docs.conda.io/en/latest/miniconda.html>`_.
+         b. Scroll down to the **Latest Miniconda Installer Links** section and click the MacOSX platform link that ends with **pkg**.
+         c. Open the downloaded file and follow through installation prompts.
 
-      .. code-block:: bash
+      2. Open a terminal window.
+      3. Install mamba on the ``base`` conda environment:
 
-         conda create -n nextstrain -c conda-forge -c bioconda \
-           augur auspice nextstrain-cli nextalign snakemake awscli git pip
+         .. code-block:: bash
 
-      Confirm that the installation worked.
+            conda install -n base -c conda-forge mamba --yes
+            conda activate base
 
-      .. code-block:: bash
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-         conda activate nextstrain
-         nextstrain check-setup --set-default
+         .. code-block:: bash
 
-      The final output from the last command should look like this:
+            mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-      .. code-block:: bash
+      5. Install the remaining Nextstrain components. There are two options:
 
-         Setting default environment to native.
+         a. Docker (recommended) – install Docker Desktop using `the official guide <https://docs.docker.com/desktop/mac/install/>`_.
+         b. Native – install all the necessary software using mamba:
 
-      Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-      Next, try :doc:`tutorials/quickstart`.
+            .. code-block:: bash
 
-   .. group-tab:: Docker
+               mamba install -c conda-forge -c bioconda augur auspice snakemake --yes
```

Comment:
In our current docs, we ask folks to install:

```
 augur auspice nextstrain-cli nextalign snakemake awscli git pip
```

The `pip` and `awscli` dependencies don't seem to be necessary anymore. Conda used to complain when you didn't explicitly install pip in your environment, but I didn't see this in my recent installation. The Nextstrain CLI's `remote` commands do most of what a Nextstrain user would need from the `awscli`.

We should keep the `nextalign` dependency, since it is in the Dockerfile, used in the ncov work, and is likely to be used in seasonal-flu and other workflows, soon. (There's also a `nextclade` conda package, but I don't know if we plan to use this as broadly yet.)

I'm not sure about dropping the `git` dependency. It's necessary for anyone to run our tutorials (until we document how to download from GitHub, for example) and novice users may not have it installed already or it might be a pain for them. I'd err on the side of keeping it, for now, unless we know that everyone already has it installed.

Reply:
Got it, I haven't tried ncov with native yet and had git preinstalled. Fixed here: https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/4584bd292626e4c6671d37e5870157369a7557dd